ckine/model.py

```diff
@@ -122,6 +122,67 @@ def runCkineUP(tps, rxntfr, preT=0.0, prestim=None, actV=None):
     return yOut
 
 
+def runCkineUmut(tps, rxntfr, mutdict, mut_name, preT=0.0, prestim=None):
+    """ Standard version of solver that returns species abundances given times and unknown rates. """
+    return runCkineUPmut(tps, np.atleast_2d(rxntfr.copy()), mutdict, mut_name, preT, prestim)
+
+
+def runCkineUPmut(tps, rxntfr, mutdict, mut_name, preT=0.0, prestim=None, actV=None):
```

Comment:
Can you just modify `runCkineUP` to accomplish this? You can make an optional argument by `mut_name=None`. It'll then be `None` if the user doesn't provide the argument, which you can check.